src/v2/schema/application.js

```diff
@@ -10,6 +10,8 @@
 // Copyright Contributors to the Open Cluster Management project
 import { gql } from 'apollo-server-express';
 
+var selfLinkNotice = "selfLink has been removed in K8s 1.20";
```

Comment:
Is this the way to add a variable - to remove duplication code smell? @jlpadilla

Reply:
Use `const` instead of `var`